### DISCUSSION TITLE
Is NextUI compatible with TailWindUI ( https://tailwindui.com/components )?

### DISCUSSION THREAD

**Discussion by biapar** [heroui-inc/heroui#1486] [Q&A]
I with to build a dashboard with NextUI.
Is't compatible with TailWindUI ( https://tailwindui.com/components )?
Thanks.

**Comment by cleytoncarvalho** [comment]
I have a related question:

Can I use @tailwindcss/forms plugin with next-ui?

**Reply by cleytoncarvalho** [reply]
Already have my answer: no.
